test/integration/serviceaccount/external_jwt_signer_test.go

```diff
@@ -235,6 +238,36 @@ func TestExternalJWTSigningAndAuth(t *testing.T) {
 	}
 }
 
+func waitForDataTimestamp(t *testing.T, client kubernetes.Interface, minimumDataTimestamp time.Time) {
```

Comment:
follow-up for 1.33 in https://github.com/kubernetes/kubernetes/pull/128967 to actually record the data timestamp from the external signer as a float instead of truncating to seconds